mava/configs/env/mabrax.yaml

```diff
@@ -0,0 +1,16 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+
+# Possible envs: [ant_4x2, halfcheetah_6x1, hopper_3x1, humanoid_9|8, walker2d_2x3]
+env_name: ant_4x2
```

Comment:
Can we please standardise this to follow what we do in all the other environments as well where `env_name: MaBrax` and then we initialise different scenarios with `scenario.task_name`. It will also make the json logs much easier to handle later on.

Reply:
Done ² 🙌